cartridges/int_bolt_embedded_sfra/cartridge/client/default/js/eventListenerRegistration.js

```diff
@@ -10,6 +10,10 @@ $(document).ready(function () {
     $('.submit-customer').attr('disabled', 'true');
     const isBoltEmbeddedExists = setInterval(async function () {
         const containerToMount = $('#email-guest').parent().get(0);
+        if (typeof containerToMount === 'undefined') {
+            clearInterval(isBoltEmbeddedExists);
+            return;
+        }
```

Comment:
what is this block used for?

Reply:
@serenayanbolt  Currently the Embedded Accounts framework on checkout page is associated with email-guest input field, but this HTML element is loaded for non-registered customers only, as a result, when login via SSO, there is a JS error "containerToMount is undefined. Then I add this block to fix this error.

Reply:
ok, I see, thanks for catching this!